LocalSettings.php

```diff
@@ -3236,6 +3236,9 @@
 			NS_FILE
 		],
 	],
+	'wgModerationIgnoredInNamespaces'=> [
```

Comment:
why does wgModerationOnlyInNamespaces not also have an entry in LocalSettings.php?

Reply:
It wasn't on LS.php before, that's why I added it via this commit.

Reply:
ManageWikiNamespaces variables don't need LS variables. This one is already in MWN.

Reply:
But looks like UO said they should not be in LS.php at all if they're going to be added to managewikinamespaces https://github.com/miraheze/mw-config/pull/5053#issuecomment-1377647676